taskcluster/docker/pushapkscript/Dockerfile

```diff
@@ -1,9 +1,30 @@
-ARG DOCKER_IMAGE_PARENT
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 FROM $DOCKER_IMAGE_PARENT
 
-VOLUME /builds/worker/checkouts
-VOLUME /builds/worker/.cache
+# %include pushapkscript
+ADD --chown=app:app topsrcdir/pushapkscript /app/pushapkscript
```

Comment:
We need to copy files from `topsrcdir/pushapkscript/docker.d` to `/app/docker.d` on the image (same for the other scripts)

Reply:
Nice find!

Reply:
Should be copied everywhere now.